### PR TITLE
Forth Culture Group PR

### PR DESCRIPTION
**Pull request by AvalonXD**

### CODE REVIEW

File: BP/common/culture/cultures/00_central_semitic.txt [hunk 1]
```diff
@@ -0,0 +1,1140 @@
...
+			10 = arab
+		}
+
+		mercenary_names = {
+			{ name = "mercenary_company_sarkans_of_place" }
+			{ name = "mercenary_company_company_of_the_oasis" }
+			{ name = "mercenary_company_desert_brethren" }
+		}
+	}
+	egyptian = {
```
Comment (loup99) [inline BP/common/culture/cultures/00_central_semitic.txt:392]: The name is misleading in the mod, while the code name can stay like that for compatibility (although I would add a comment to say Egyptian Arabic directly here too) we should overwrite in localisation

File: BP/common/culture/cultures/00_central_semitic.txt [hunk 2]
```diff
@@ -0,0 +1,1140 @@
...
+		ethnicities = {
+			10 = arab
+		}
+
+		mercenary_names = {
+			{ name = "mercenary_company_the_wanderers" }
+			{ name = "mercenary_company_bedouin_company_of_place" }
+		}
+	}
+	levantine = {
```
Comment (loup99) [inline BP/common/culture/cultures/00_central_semitic.txt:211]: This name is misleading as well, same as for Egyptian, Levantine isn't necessarily associated with Arabic.

File: BP/history/characters/romano_frank.txt [hunk 3]
```diff
@@ -31,7 +31,9 @@
 	culture="romano_frank"
 	father=1310003
 	410.1.1={
+		birth=yes
 	}
 	460.1.1={
+		death=yes
```
Comment (loup99) [inline BP/history/characters/romano_frank.txt:37]: I don't know how that got added without the birth and date effects, going to fix for CK2 as well

File: BP/common/culture/cultures/00_chinese.txt [hunk 4]
```diff
@@ -0,0 +1,577 @@
...
+		dynasty_title_names = no
+
+		ethnicities = {
+			10 = asian
+		}
+	}
+
+	# WTWSMS
+
+	chinese = {
```
Comment (loup99) [inline BP/common/culture/cultures/00_chinese.txt:265]: I think Han is actually better here, isn't this one redundant? We integrated it from VIET before vanilla had added Han.

File: BP/common/culture/cultures/00_east_iranian.txt [hunk 5]
```diff
@@ -0,0 +1,795 @@
...
+		
+		patronym_suffix_male = "dynnpat_suf_y_fyrt"
+		patronym_suffix_female = "dynnpat_suf_y_chyzg"
+		
+		ethnicities = {
+			7 = arab
+			6 = mediterranean
+		}
+	}
+	khwarezmian = { # WTWSMS
```
Comment (loup99) [inline BP/common/culture/cultures/00_east_iranian.txt:215]: If this is a culture the mod added you can group it together with the others below, no?

File: BP/common/culture/cultures/00_magyar.txt [hunk 6]
```diff
@@ -0,0 +1,428 @@
+﻿magyar_group = {
```
Comment (loup99) [inline BP/common/culture/cultures/00_magyar.txt:1]: Has anything been modified in this file from vanilla? If no we don't need it in the mod?

File: BP/common/culture/cultures/00_chinese.txt (hunk 4 above)
Comment (AvalonXD) [reply BP/common/culture/cultures/00_chinese.txt:265]: Yeah

File: BP/common/culture/cultures/00_east_iranian.txt (hunk 5 above)
Comment (AvalonXD) [reply BP/common/culture/cultures/00_east_iranian.txt:215]: All that I changed was the "patronym_prefix_female". Everything else is identical to vanilla like you did for Roman in the Latin culture group.

File: BP/common/culture/cultures/00_magyar.txt (hunk 6 above)
Comment (AvalonXD) [reply BP/common/culture/cultures/00_magyar.txt:1]: No, nothing has

File: BP/common/culture/cultures/00_east_iranian.txt (hunk 5 above)
Comment (loup99) [reply BP/common/culture/cultures/00_east_iranian.txt:215]: Ok

**Review by loup99 — APPROVED** [review]
Great work with this pull request, I don't personally have any more comments, but I think it can be left open until the end of the week in case anyone else has feedback.

**Pull request merged**